index.html

```diff
@@ -159,7 +159,7 @@ <h3>Japanese Text Support</h3>
           <tr>
             <td><code>#textEmphasis-minimal</code></td>
 
-            <td>none</td>
+            <td>Only <code>outside</code> position semantics are required.</td>
```

Comment:
This is ambiguous: it could mean

1. only position semantics are required, and only the value `outside` needs to be supported, OR 
2. of the position semantics, only the value `outside` needs to be supported, but `emphasis-style` does also need to be supported, barring quoted strings as excluded by `#textEmphasis-minimal`

Please could you clarify this @palemieux ?

Reply:
I would say (2).

Reply:
Will correct to _of position semantics, only <outside> is supported_